### PR TITLE
Fix unescape not html entities

### PR DESCRIPTION
**Pull request by apinrdw**
it's encode the '& lt;' to '<' that already opened issue on #288

### CODE REVIEW

**Comment by hgani** [comment]
Hi,

Can this pull request be merged? Or is there any issue?

Thanks

File: app/models/mailboxer/notification.rb [hunk 1]
```diff
@@ -162,8 +162,9 @@ def mark_as_deleted(participant)
 
   #Sanitizes the body and subject
   def clean
-    self.subject = sanitize(subject) if subject
-    self.body    = sanitize(body)
+    self.subject   = sanitize(subject) if subject
+    sanitized_body = String.new sanitize(body)
```
Comment (jcoyne) [inline app/models/mailboxer/notification.rb]: There is no need for String.new here. That's just causing an unnecessary object creation.

File: app/models/mailboxer/notification.rb [hunk 2]
```diff
@@ -162,8 +162,9 @@ def mark_as_deleted(participant)
...
+    self.body      = CGI.unescapeHTML(sanitized_body)
```
Comment (jcoyne) [inline app/models/mailboxer/notification.rb]: The unescapeHTML is just reverting what was done by `sanitize(body)`, right? Why bother doing anything then?

**Comment by jcoyne** [comment]
Sorry for being inattentive to this gem. Thank you for your contribution.  Please add a test to this pull request.  It should show that any script injection can be handled by other parts of the stack.  Also, please squash the commits into a single commit.

**Pull request closed without merging**